### PR TITLE
Add README.md, various other improvements

### PR DESCRIPTION
**Pull request by yjhn**
## Makefile

- since `biber` [is included in `texlive-full` Ubuntu package](https://packages.ubuntu.com/kinetic/texlive-full), it does not need to be installed separately
- add `.PHONY` recipe (see [this](https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile) for explanation)
- `clean` will only remove files in `.gitignore`
- stop (cancel) PDF generation on error

## README.md

- add README with basic installation and PDF generation instructions (haven't checked them on Windows and macOS, but they should be at least mostly correct)

## VUMIFPSkursinis.cls

- add more comments
- fix PDF bookmarks
- silence useless warnings
- add ability to remove signature places in title page (course works are not printed anymore anyway)
- move city and year to end of title page (it is now fixed in place and will only move if the title page content overflows to another page)
- delete usages of `\makeatletter` and `\makeatother` as [they are not needed in class files](https://tex.stackexchange.com/questions/62583/is-it-really-bad-to-use-makeatletter-and-makeatother-in-a-package-or-class-fil)

## kursinis.tex

- signature places turned off by default
- add [`microtype`](https://ctan.org/pkg/microtype) package for slightly enhanced look

### CODE REVIEW

File: kursinis.tex [hunk 1]
```diff
@@ -27,7 +27,7 @@
 \author{Vardenis Pavardenis}
 % \secondauthor{Vardonis Pavardonis}   % Pridėti antrą autorių
 \supervisor{prof. habil. dr. Vardaitis Pavardaitis}
-\addsignatureplaces{} % prideda parašų vietas tituliniame puslapyje
+% \addsignatureplaces{} % prideda parašų vietas tituliniame puslapyje
```
Comment (albertas) [inline kursinis.tex:31]: Ar nebūtų geriau, jeigu parašų vietos būtų pridėtos pagal numatymą?

**Review by albertas — APPROVED** [review]

File: kursinis.tex (hunk 1 above)
Comment (yjhn) [reply kursinis.tex:31]: Gal, bet dabar bent jau kursiniai darbai nebėra spausdinami, tai kaip ir beprasmiška šitą turėti.

Comment (yjhn) [reply kursinis.tex:31]: Nors nurodymuose pateiktame pavyzdyje parašų vietos vis dar yra.

Comment (albertas) [reply kursinis.tex:31]: Tai gal kursiniam darbui galime nuimti, o bakalauriniam darbui kol kas palikti parašus įjungtus? Manau būtų geras kompromisas.

**Review by albertas — APPROVED** [review]

**Comment by yjhn** [comment]
Ko laukiam?

**Pull request merged**